[MTM-61063] added update for documentation for TS migration (#2548)
SoftwareAG/c8y-docs: content/enterprise-tenant/enhanced-time-series-support.md

Migration and seamless configuration will be part of future releases.

{{< c8y-admon-important >}}

> Once enabled, do not switch this property back to `DISABLED` because it may cause data loss and integration issues with third-party systems.

Reviewer: ```suggestion
Once enabled, do not switch this property back to `DISABLED` as it may cause data loss and integration issues with third-party systems.
```

> Once enabled, avoid switching the property back to `DISABLED` as this can lead to experience data loss. Switching the property back to `DISABLED` results in integration issues with third-party systems.

Reviewer 2: Maybe:
```suggestion
Once enabled, do not switch this property back to `DISABLED` because it may cause data loss and integration issues with third-party systems.
```
?

> **Once enabled, avoid switching the property back to `DISABLED` as this can lead to experience data loss. Switching back to `DISABLED` generates integration issue with third part systems. Do this only in case of an issue or emergency.**

Reviewer 3: ```suggestion
Once enabled, avoid switching the property back to `DISABLED` as this can lead to experience data loss. Switching the property back to `DISABLED` results in integration issues with third-party systems. Do this only in case of an issue or emergency.
```

Reviewer 3: You do not need to highlight this text in bold as it is already an "Important" note.

> Once enabled, avoid switching the property back to `DISABLED` as this can lead to experience data loss. Switching the property back to `DISABLED` results in integration issues with third-party systems. Do this only in case of an issue or emergency.

Reviewer 4: It would be better to not suggest possibility of doing switch back under any circumstances.

```suggestion
Once enabled, avoid switching the property back to `DISABLED` as this can lead to experience data loss. Switching the property back to `DISABLED` results in integration issues with third-party systems.
```
